actors/builtin/multisig/multisig_actor.go

```diff
@@ -120,8 +117,14 @@ func (a *MultiSigActor) Propose(rt vmr.Runtime, params *ProposeParams) TxnID {
 		Value:      params.Value,
 	}
 
-	st.PendingTxns[txnID] = txn
 	st.PendingApprovals[txnID] = autil.ActorIDSetHAMT_Empty()
+
+	pendingTxnCID, err := setPendingTxn(context.TODO(), rt, txnID, txn)
```

Comment:
The name pendingTxnCID is confusing. This is assigning over the HAMT root node, so should be `newPendingTxnsRoot` or similar.